packages/x-data-grid/src/models/params/gridCellParams.ts

```diff
@@ -65,6 +65,10 @@ export interface GridCellParams<
    * the tabIndex value.
    */
   tabIndex: 0 | -1;
+  /**
+   * The api of `DataGrid`.
```

Comment:
```suggestion
   * GridApi that let you manipulate the grid.
```

For consistency with the other `api` annotations.